packages/block-library/src/pullquote/blockquote.native.js

```diff
@@ -12,15 +12,23 @@ import { Children, cloneElement } from '@wordpress/element';
 import styles from './blockquote.scss';
 
 export const BlockQuote = ( props ) => {
+	const citationStyle = { ...styles.citation };
+	const quoteStyle = { ...styles.quote };
+
+	if ( props.textColor ) {
+		quoteStyle.color = props.textColor;
+		citationStyle.color = props.textColor;
```

Comment:
Let's add the placeholder color as well =)

```suggestion
		quoteStyle.color = props.textColor;
		quoteStyle.placeholderColor = props.textColor;
		citationStyle.color = props.textColor;
		citationStyle.placeholderColor = props.textColor;
```